clearpath_robot/services/clearpath-platform-extras.service

```diff
@@ -7,6 +7,8 @@ After=clearpath-robot.service
 User=administrator
 Type=simple
 ExecStart=/usr/sbin/clearpath-platform-extras-start
+LimitRTPRIO=infinity
+LimitMEMLOCK=infinity
```

Comment:
Instead of allowing infinite memory, on the outside chance that there's a memory leak in a node implementation is it worth setting a finite, but still very high limit?